src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -5000,6 +5000,34 @@ def test_vmss_orchestration_mode(self, resource_group):
             self.check('platformFaultDomain', 0)
         ])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_vmss_orchestration_mode_', location='eastus2euap')
+    @VirtualNetworkPreparer(location='eastus2euap')
+    def test_vmss_complex_orchestration_mode(self, resource_group, virtual_network):
+
+        self.kwargs.update({
+            'vmss': 'vm-ultrassd',
+            'vmss2': 'vm-ultrassd2',
+            'vnet_name': virtual_network
+        })
+
+        self.cmd('vmss create -g {rg} -n {vmss} --orchestration-mode Flexible --single-placement-group false '
+                 '--platform-fault-domain-count 1 --vm-sku Standard_DS1_v2 --instance-count 0 --admin-username testvmss '
+                 '--admin-password This!s@Terr!bleP@ssw0rd --computer-name-prefix testvmss --image debian '
+                 '--vnet-name {vnet_name} --subnet default --network-api-version 2020-11-01')
```

Comment:
`--network-api-version 2020-11-01`, is it required by service team? Azure CLI usually selects a API version for users intelligently.

Reply:
yes, as requested in  https://github.com/Azure/azure-cli/issues/17830